src/widgets/option-item/ui/option-item.module.scss

```diff
@@ -0,0 +1,22 @@
+.option-item {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  max-width: 380px;
+
+  color: $color-gray-600;
```

Comment:
It's useless because this color is default

Reply:
Done